src/dc_imex.rs

```diff
@@ -333,15 +334,15 @@ pub unsafe fn dc_decrypt_setup_file(
     context: &Context,
     passphrase: *const libc::c_char,
     filecontent: *const libc::c_char,
-) -> *mut libc::c_char {
+) -> Option<String> {
```

Comment:
I think it makes sense to switch to `Result` directly, if changing the return type